hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/7_4_0/xxxx-add-support-for-head-request.yaml

```
@@ -0,0 +1,4 @@
+---
+type: add
+issue: xxxx
```

```suggestion
issue: 6152
```

Can you also please rename this file to 6152? It's fine to use a PR number in place of a ticket number, but it does need a number or it breaks our doc builder.